app/tournament/match.js

```diff
@@ -0,0 +1,32 @@
+export class Match {
+    constructor(round) {
+        this.round = round;
+        this.players = [];
+    }
+
+    addPlayer(player) {
+        if (this.players.length >= 2) {
+            throw new Error("Match has already 2 players");
+        }
+        this.players.push(player);
+    }
+
+    setNextMatch(match) {
+        this.nextMatch = match;
+    }
+
+    setWinner(playerIndex) {
+        if (this.winner === undefined) {
```

Comment:
Don´t fail silently. If winner is already defined, calling this method is a bug.